pkg/informer/supportedResources.go

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) 2020 Red Hat, Inc.
```

Comment:
Remove this line

Reply:
Every script in this project has that in the header.

Reply:
We used to add that before. But after we open-sourced, the recommendation is not to have that. Since, this is a new file it should only have  

> // Copyright Contributors to the Open Cluster Management project

Reply:
Gotcha, will change this file here and open PR for the others in this repo.